_targets/build.project.armv7a7-imx6ull

```diff
@@ -36,39 +34,46 @@ export PORTS_CURL=y
 #
 SIZE_PAGE=$((0x1000))
 PAGE_MASK=$((~(SIZE_PAGE - 1)))
+KERNEL_OFFS=$((0x0))          # Disk image kernel offset (in bytes)
+ADDR_DDR=$((0x80000000))
+
+
+#
+# Project specific build
+#
 
+BOOT_DEVICE="flash0"              # Default boot device
+NAME_USER_SCRIPT="user.plo"       # User script name
+MAGIC_USER_SCRIPT=$((0xdabaabad)) # User script magic value
+OFFS_USER_SCRIPT=$((0x0))         # Disk image user script offset (in bytes)
+OFFS_ROOTFS=$((0x1000))           # Disk image rootfs offset (in 512 byte sectors)
 
 
 # Pre-init script is launched before user script
 PREINIT_SCRIPT=(
-	"map kddr 0x80000000 0x8002ffff rwx"
-	"map ddr 0x80030000 0x90000000 rwx"
+	"map ddr 0x80000000 0x87ffffff rwx"
 	"phfs usb0 1.2 phoenixd"
 	"console 0.0"
 	"wait 2000"
-	"kernel usb0"
-	"app usb0 -x dummyfs ddr ddr"
-	"app usb0 -x imx6ull-uart ddr ddr"
-	"app usb0 -x psh ddr ddr"
 )
 
 
-# Obligatory applications to run Phoenix-RTOS
-USER_SCRIPT=(
-	"wait 2000"
+#TODO: if plo is introduced, BOOT_DEVICE should be changed
+UPLOAD_SCRIPT=(
+	"kernelimg ${BOOT_DEVICE}"
+	"app ${BOOT_DEVICE} -x dummyfs ddr ddr"
+	"app ${BOOT_DEVICE} -x imx6ull-uart ddr ddr"
+	"app ${BOOT_DEVICE} -x imx6ull-flash;-p;0:8;-p;8:16;-p;24:16;-p;64:128;-p;192:128;-p;2048:1952;-p;4032:16 ddr ddr"
+	"app ${BOOT_DEVICE} -x psd;/dev/flash0;/dev/flash0p1;/dev/flash0p2;/dev/flash0p3;/dev/flash0p4;/dev/flash0p5;/dev/flash0p6;/dev/flash0p7 ddr ddr"
 )
 
-
-#
-# Project specific build
-#
-
-BOOT_DEVICE="flash0"          # Default boot device
-NAME_USER_SCRIPT="user.plo"   # User script name
-MAGIC_USER_SCRIPT=            # User script magic value
-OFFS_USER_SCRIPT=$((0x12000)) # Disk image user script offset (in bytes)
-KERNEL_OFFS=$((0x13000))      # Disk image kernel offset (in bytes)
-OFFS_ROOTFS=$((0x1000))       # Disk image rootfs offset (in 512 byte sectors)
+PRIMARY_SCRIPT=(
+	"kernelimg ${BOOT_DEVICE}"
+	"app ${BOOT_DEVICE} -x dummyfs;-N;devfs;-D ddr ddr"
+	"app ${BOOT_DEVICE} -x imx6ull-uart ddr ddr"
+	"app ${BOOT_DEVICE} -x imx6ull-flash;-p;0:8:fcb;-p;8:16:kernel1;-p;24:16:kernel2;-p;64:128:rootfs1;-p;192:128:rootfs2;-p;2048:1952:data;-p;4032:16:local;-r;jffs2:4:5 ddr ddr"
+	"app ${BOOT_DEVICE} -x psh ddr ddr"
+)
```

Comment:
It's ok, but I think this particular config should be done in DTR project's image build - in phoenix-rtos-project we want example project - so dummyfs based system should be sufficient.

Reply:
Ok, I've added changes. Primary image consists of `dummfs`, `imx6ull-uart` and `psh`.